packages/ng/core-select/input/select-input.component.ts

```diff
@@ -124,6 +125,13 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 	@Input() optionComparer: LuOptionComparer<TOption> = (option1, option2) => JSON.stringify(option1) === JSON.stringify(option2);
 	@Input() optionKey: (option: TOption) => unknown = (option) => option;
 
+	noClueIcon = input(false, { transform: booleanAttribute });
```

Comment:
```suggestion
	noClueIcon = input<boolean, boolean>(false, { transform: booleanAttribute });
```

Reply:
Adding that would make things more complicated, input types should be properly inferred here.